src/slack_notify.ml

```diff
@@ -66,16 +70,22 @@ let execute token username channel icon_url icon_emoji msg =
   let channel = Slacko.channel_of_string channel in
   let chat = Slacko.Channel channel in
   let msg = Slacko.message_of_string msg in
+  let attachments =
+    match atxt with
+    | None -> None
+    | Some txt -> Some ([Slacko.attachment ~text:txt ()])
+  in
   Slacko.chat_post_message token chat
     ?username:(username)
     ?icon_emoji:(icon_emoji)
     ?icon_url:(icon_url)
+    ?attachments:(attachments)
     msg
   >|= (fun c ->
     print_endline @@ string_or_bust c)
   |> Lwt_main.run
 
-let execute_t = Cmdliner.Term.(pure execute $ token $ username $ channel $ icon_url $ icon_emoji $ message)
+let execute_t = Cmdliner.Term.(pure execute $ token $ username $ channel $ icon_url $ icon_emoji $ attachment $ message)
```

Comment:
At this point it would make sense to wrap this line (actually this should've happened before but as we're changing the line anyway).